### PR TITLE
Sannas weather app

### PR DESCRIPTION
**Pull request by sannadw**
**Link to my deployed project:** 
https://sannasweatherapp.netlify.app/

**My reflections on how this week's project turned out:**
Im happy I got the API to work, but I wish I had much more time to make it look better. 

**Things I'd like to have clarified or explained in more detail:**
My code does not look very clean. When showing different images depending on the weather I added a lot of code in my if else statement. Im sure this can be done in a prettier way.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                    rainImage.classList.add("visible");
+                    rainImage.classList.remove("hidden");
+                    sunImage.classList.add("hidden");
+                    sunImage.classList.remove("visible");
+                    cloudImage.classList.add("hidden");
+                    cloudImage.classList.remove("visible");
+                }
+            }
+
+            setWeatherImages()
```
Comment (rebkova) [inline script.js:55]: Super clean and easy to understand.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                    rainImage.classList.add("hidden");
+                    rainImage.classList.remove("visible");
+                } else if (weatherType === 'Clear') {
+                    sunImage.classList.add("visible");
+                    sunImage.classList.remove("hidden");
+                    cloudImage.classList.add("hidden");
+                    cloudImage.classList.remove("visible");
+                    rainImage.classList.add("hidden");
+                    rainImage.classList.remove("visible");
+                } else if (weatherType === 'Rain' || weather.main === 'Drizzle') {
```
Comment (rebkova) [inline script.js:45]: Any specific reason why you used weather.main instead of weatherType in the second operand?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+  <title>Weather app</title>
+</head>
+
+<body>
+  <main>
+
+    <div class="tablet-weatherandimage">
```
Comment (rebkova) [inline index.html:15]: To follow the naming convention, you could write the class name as: tablet-weather-and-image.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css" />
```
Comment (rebkova) [inline index.html:7]: Very nice, including ./ in your paths! :)

**Review by rebkova — COMMENTED** [review]
Sanna, your code is clean and easy to read. You know how to fetch the data from an API and display it in a required format. In addition, your page is responsive. Nice job on this week's project.  Keep it up!

File: index.html (hunk 4 above)
Comment (EmOlsz) [reply index.html:7]: :)

File: script.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
+const API_KEY = "f54ca9831c8974c87fd4826fae420a1a"
+const API_URL = `https://api.openweathermap.org/data/2.5/weather?q=Malmo,Sweden&units=metric&APPID=${API_KEY}`
+const API_URL_5DAYS = `https://api.openweathermap.org/data/2.5/forecast?q=Malmo,Sweden&units=metric&APPID=${API_KEY}`
+
+
+fetch(API_URL)
+    .then((response) => {
```
Comment (EmOlsz) [inline script.js:7]: No need to wrap `response` in extra pair of parentheses, since it's only one argument we can omit them

File: script.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+            weatherContainer.innerHTML += `<p>sunset ${sunSetTime}</p>`
+
+            // WEATHER IMAGE DEPENDING ON WEATHER
+            const cloudImage = document.getElementById('cloud-image')
+            const sunImage = document.getElementById('sunglasses-image')
+            const rainImage = document.getElementById('umbrella-image')
+            const weatherType = weather.main
+            const setWeatherImages = () => {
+                if (weatherType === 'Clouds') {
+                    cloudImage.classList.add("visible");
```
Comment (EmOlsz) [inline script.js:32]: We shouldn't design our code logic like that - there is just too much of code.
You could just have one HTML element (maybe image?) and pass to it new value as `src`. Thanks to that we won't have to write 6 lines of code in each statement.
This needs to be changed

File: script.js [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+            setWeatherImages()
+
+        });
+
+    })
+
+
+
+
```
Comment (EmOlsz) [inline script.js:63]: There is a lot of places like this in your code - please, go through whole project and delete empty lines

File: style.css [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+.forecast-ul {
+    display: flex;
+    justify-content: space-between;
+}
+
+li {
+    border-bottom: 1px dashed;
+    padding-bottom: 4px;
+    padding-top: 4px;
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply style.css:36]: Haha, good :D

File: style.css [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+    font-size: 23px;
+    list-style-type: none;
+}
+
+li:last-child {
+    border-bottom: 0px;
```
Comment (EmOlsz) [inline style.css:42]: `border-bottom: 0;`

File: style.css [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+    justify-content: space-around;
+}
+
+}
+  
+/* MEDIA QUERIES */
+/* På skärmar större än 1025 px */
+@media screen and (min-width: 1025px) {
+   
+#city {
```
Comment (EmOlsz) [inline style.css:91]: We shouldn't style by id's, let's change it to class

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Good job this week Sanna! Your app looks very nice, code in general is also well done :)
Still, there are some things that needs to be fixed. Take a look on my comments and provide changes, after that I will be able to approve your project!

**Pull request closed without merging**